docs/user-guide/longitudinal-profile.md

```diff
@@ -0,0 +1,87 @@
+# Longitudinal Profile
+
+*******************
+
+The **Longitudinal Profile** tool allows the user to analyze the altitude or depth data present in digital terrain models or bathymetric surveys and displays the found values in an interactive graphic profile.
+
+!!! note
+    By default, the **Longitudinal Profile** plugin is ready to be configured for [application contexts](application-context.md#configure-plugins), and is not available in the default plugin configuration.
+    The plugin works only if the [WPS process](gs-wps-longitudinal-profile-2.24-SNAPSHOT.zip) is configured on the [GeoServer](http://geoserver.org/).
+
+By clicking the **Longitudinal Profile** <img src="../img/button/long-profile-button.jpg" class="ms-docbutton"/> button, available in the [Side Toolbar](mapstore-toolbars.md#side-toolbar), a drop down menu opens and the user can select the way to calculate altitude and distance traveled values, choosing between three options:
```

Comment:
```suggestion
By clicking the **Longitudinal Profile** <img src="../img/button/long-profile-button.jpg" class="ms-docbutton"/> button, available in the [Side Toolbar](mapstore-toolbars.md#side-toolbar), a drop down menu opens so that the user can manage the available options including different ways to calculate the profile: